core/chain/coin/price/solana/getSolanaTokenUSDValue.ts

```
@@ -0,0 +1,63 @@
+    //"https://quote-api.jup.ag/v6/quote?inputMint=\(inputMint)&outputMint=\(outputMint)&amount=\(amount)&slippageBps=\(slippageBps)"
+    // Process each request sequentially to avoid rate limits
+    for (const coin of coins.filter((coin) => coin.id !== "SOL")) {
+      const url = generateSolanaTokenQuoteUrl(
+        coin.id,
+        SOLANA_USDC_MINT,
+        amountDecimal.toString(),
+        "50"
+      );
+
+      try {
+        const response = await fetch(url);
```

ESLint

[error] Delete `;`

(prettier/prettier)